es-fork.yaml

```diff
@@ -0,0 +1,79 @@
+title: "op-geth - go-ethereum fork diff overview"
+footer: |
+  Fork-diff overview of [`SWC op-geth`](https://github.com/ethstorage/op-geth), a fork of [`official op-geth`](https://github.com/ethereum-optimism/op-geth).
+  and execution-engine of the [SWC OP-stack](https://github.com/ethstorage/optimism).
+base:
+  name: official op-geth
+  url: https://github.com/ethereum/go-ethereum
```

Comment:
I am curious that why is geth used as the base instead of op-geth?

Reply:
Good catch! Fixed.